src/pages/[platform]/build-a-backend/data/aws-appsync-apollo-extensions/index.mdx

```diff
@@ -38,32 +37,42 @@ To learn more about Apollo, see https://www.apollographql.com/docs/kotlin.
 
 </InlineFilter>
 
-## Features
+## Install the AWS AppSync Apollo Extensions library
 
-AWS AppSync Apollo Extensions provide AWS AppSync authorizers to be used with the Apollo client to make it simple to apply the correct authorization payloads to your GraphQL operations.
+<InlineFilter filters={["android"]}>
 
-The Amplify library provides components to facilitate configuring the authorizers with Apollo client by providing configuration values to connect to your Amplify Data backend.
+<BlockSwitcher>
 
-### Install the AWS AppSync Apollo Extensions library
+<Block name=" Amplify Data">
 
-<InlineFilter filters={["android"]}>
+Add the `apollo-appsync-amplify` dependency to your app/build.gradle.kts file.
+
+```kotlin title="app/build.gradle.kts"
+dependencies {
+    // highlight-start
+    // Connect Apollo to AppSync, delegating some implementation details to Amplify
+    implementation("com.amplifyframework:apollo-appsync-amplify:1.0.0")    
+    // highlight-end
+}
+```
+
+</Block>
 
-To connect Apollo to AppSync without using Amplify, add the `apollo-appsync` dependency to your app/build.gradle.kts file. If your
-application is using Amplify Android, add the `apollo-appsync-amplify` dependency instead.
+<Block name="AppSync">
```

Comment:
AppSync should be "AWS AppSync" throughout.

But how about making these tabs "With Amplify" and "Without Amplify"?

Reply:
Yeah I like that better.